phoenix-core/src/it/java/org/apache/phoenix/rpc/UpdateCacheIT.java

```diff
@@ -69,6 +70,10 @@
 @Category(ParallelStatsDisabledTest.class)
 public class UpdateCacheIT extends ParallelStatsDisabledIT {
 
+    private boolean isLastDDLTimestampValidationEnabled
+            = QueryServicesOptions.DEFAULT_LAST_DDL_TIMESTAMP_VALIDATION_ENABLED
```

Comment:
Why not to get the last ddl timestamp validation enabled flag from the connection properties?

Reply:
will use config